charts/pxc-database/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for installing Percona XtraDB Cluster Databases using the PXC Operator.
-name: pxc-database
+name: pxc-db
 home: https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html
 version: 0.1.4
```

Comment:
always bump the chart version on a change using semver scheme

Reply:
ok